atomic_reactor/dirs.py

```diff
@@ -253,3 +253,18 @@ def for_all_platforms_copy(self, action: FileCreationFunc) -> List[Path]:
                     copyfile(src_file, dest)
 
         return the_new_files
+
+
+class SourceBuildDir(RootBuildDir):
```

Comment:
This kinda sounds like a combination of the `Source` and `BuildDir` classes (which it isn't). Maybe rename it to `SourceContainersBuildDir`? Or maybe it doesn't need to be a subclass, having the attributes in RootBuildDir might be OK if they have "source_container" in the name